src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -704,7 +712,26 @@ def create_policy_assignment(policy, name=None, display_name=None, params=None,
             params = shell_safe_json_parse(params)
 
     PolicyAssignment = get_sdk(ResourceType.MGMT_RESOURCE_POLICY, 'PolicyAssignment', mod='models')
-    assignment = PolicyAssignment(display_name, policy_id, scope, params if params else None)
+    assignment = PolicyAssignment(display_name, policy_id, scope)
+    assignment.parameters = params if params else None
+
+    if supported_api_version(ResourceType.MGMT_RESOURCE_POLICY, min_api='2017-06-01-preview'):
+        if not_scopes:
+            kwargs_list = []
+            for id_arg in not_scopes.split(' '):
+                if parse_resource_id(id_arg):
+                    kwargs_list.append(id_arg)
+                else:
+                    logger.error('az policy assignment create error: argument --not-scopes: \
+                    invalid notscopes value: \'%s\'' % id_arg)
```

Comment:
Same here with the use of `\`.

Right now, you are doing:
```
'My-first-line-of-text\
My-second-line-of-text'
```
Rather, it should be:
```
'My-first-line-of-text'\
'My-second-line-of-text'
```